.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
- [ ] Please fix these errors I can see in the dev tools the user can't see anything because of them
![image](https://user-images.githubusercontent.com/74744787/225732814-af72e625-7e9c-4084-ae6d-8cc7277bcfe5.png)

Reply:
Hi, @MeqdamAlqudah I think it was the issue with my Api server..... But now it is working. 👍